subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -1,14 +1,14 @@
 # Version values used in build-init templates
-# Generated file, please to not edit
+Generated file, please to not edit
```

Comment:
Not changed on purpose?

Reply:
Oops, I didn't notice that

Reply:
And there seems to be a typo "Generated file, please **do** not edit" ?